### PR TITLE
small bug (typo) fix

### PR DESCRIPTION
**Pull request by mrahanjam**
This PR fixes a typo in `CdfStudioActions` class.

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java [hunk 1]
```diff
@@ -148,7 +148,12 @@ public static WebElement locateSourceEndpointInCanvas(String sourcePluginName) {
 
   public static WebElement locateSinkNodeInCanvas(String sinkPluginName) {
     return SeleniumDriver.getDriver().findElement(
-      By.xpath("//*[contains(@data-cy,'plugin-node-" + sink + "') and @data-type='batchsink']"));
+      By.xpath("//*[contains(@data-cy,'plugin-node-" + sinkPluginName + "') and @data-type='batchsink']"));
```
Comment (itsankit-google) [inline src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java:151]: wonder how it was working earlier?
@sawantpritam are we using this method?

Comment (mrahanjam) [reply src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java:151]: It gave me an `Unable to Locate Element` error when tried using it, that's how I found the bug.

**Review by itsankit-google — APPROVED** [review]

File: src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java [hunk 2]
```diff
@@ -355,6 +355,21 @@ public static void connectSourceAndSink(String source, String sourceTitle, int i
...
   }
 
+  /**
+   * Connect two plugin nodes
+   *
+   * @param source     Source Plugin name
+   * @param target     Target Plugin name
+   * @param targetType Target Plugin name
+   */
+  public static void connectTwoPluginNodes(String source, String target, String targetType) {
```
Comment (itsankit-google) [inline src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java]: param names source and target tend to confuse with source and sink plugins, can we rename them to fromPlugin and toPlugin?

Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java]: We already have generic method to connect plugins - `establishConnection`

https://github.com/cdapio/cdap-e2e-tests/blob/25c2fd8561af42f9472f9fd7a9cee980e5bcefac/src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java#L372

File: src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java (hunk 1 above)
Comment (sawantpritam) [reply src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java:151]: We are using `locateSinkNodeInCanvas` only when `data-type='batchsink'`
We have generic locator `locateSinkNodeInCanvas` which can locate any type of plugin node using title.

https://github.com/cdapio/cdap-e2e-tests/blob/25c2fd8561af42f9472f9fd7a9cee980e5bcefac/src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java#L211

File: src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java (hunk 2 above)
Comment (mrahanjam) [reply src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java]: Addressed. Does the `Build e2e tests` workflow have to pass before merging? Seems like it's been consistently failing lately.

Comment (itsankit-google) [reply src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java]: yeah, ideally it should pass before merge. Currently it is failing due to some known issue in running preview in sandbox.

File: src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java (hunk 1 above)
Comment (mrahanjam) [reply src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java:151]: Right, I saw this but I wanted to stay away from using plugin title. I can remove my helper here and use it as a local helper in UI repo. I'm using `data-testid` to locate nodes which relies on plugin type. It's more for consistency with the rest of our tests. If you think it's for our specific use case, doesn't have to be added here.

Comment (mrahanjam) [reply src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java:151]: The more I look at it, I think these are better suited for our (UI) own use. I can easily utilize some of the methods in `ElementHelper` class to get what I want. This PR will just be limited to that tiny bug fix.

**Review by itsankit-google — APPROVED** [review]

**Pull request merged**